test/tests.js

```diff
@@ -1,6 +1,7 @@
 import jsep from "../src/jsep.js";
 
 (function() {
+/* eslint-disable brace-style */
 var binops = {
 	"+" : function(a, b) { return a + b; },
```

Comment:
Unpopular opinion: If your linting rules force you to use comments to avoid warnings, then either the code or the linting rules need to be revised. 
Thankfully, I think in this case we can just refactor the code to use arrow functions, which would be more readable too, this kind of thing is exactly why they exist.